internal/build/build.go

```diff
@@ -254,7 +254,7 @@ func buildAllPkgs(ctx *context, initial []*packages.Package, verbose bool) (pkgs
 			cl.ParsePkgSyntax(ctx.prog, pkg.Types, pkg.Syntax)
 			pkg.ExportFile = ""
 		case cl.PkgLinkIR, cl.PkgLinkExtern, cl.PkgPyModule:
-			if isPkgInLLGo(pkg.PkgPath) {
```

Comment:
The `isPkgInLLGo` func itself can also be removed now.

Reply:
remove func isPkgInLLGo